### ISSUE TITLE
Can I train and inference any other information other than cls, xywh?

### ISSUE THREAD

**Issue by kanjitree** [ultralytics/yolov5#8593]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I would like to deduce some additional information. For example, I want to inference the box for the rear and side of the vehicle.

`cls, xywh(full box), xywh(rear box), xywh(side box)`

![image](https://user-images.githubusercontent.com/109414671/179349914-8409e57c-f95a-4017-98cf-63bc72c5b4f0.png)

I already know, there is also a way to divide the super categories using softmax like yolo9000. But, this is just a way to divide a class. (reference https://github.com/ultralytics/yolov5/issues/6027)
Is there any good way?

**Comment by glenn-jocher** [comment]
@JackJepp yes, there are ways to customize YOLO for this, though it's not trivial. First and foremost you need labelled data with all the features you want to predict, then you need to update the dataloaders, loss function, validation and visualizations/loggers to your custom requirements.
